src/EnergyPlus/PhaseChangeModeling/HysteresisModel.cc

```diff
@@ -95,12 +95,12 @@ namespace Material {
             Tau2 = this->deltaTempMeltingHigh;
             if (updatedTempTDT < TempLowPCM) {
                 phaseChangeState = Phase::Crystallized;
-            } else if (updatedTempTDT >= TempLowPCM && updatedTempTDT <= TempHighPCM) {
+            } else if (updatedTempTDT <= TempHighPCM) {
```

Comment:
[src/EnergyPlus/PhaseChangeModeling/HysteresisModel.cc:98]:(style),[knownConditionTrueFalse],Condition 'updatedTempTDT>=TempLowPCM' is always true